internal/builder/packages.go

```diff
@@ -55,6 +55,24 @@ func FindBuildPackagesDirectory() (string, bool, error) {
 	return "", false, nil
 }
 
+// FindBuildPackagesDirectory method locates the target build directory for packages.
+func FindBuildPackagesDirectory() (string, bool, error) {
+	buildDir, found, err := FindBuildDirectory()
+	if err != nil {
+		return "", false, err
+	}
+
+	if found {
+		path := filepath.Join(buildDir, "integrations") // TODO add support for other package types
+		fileInfo, err := os.Stat(path)
+		if err == nil && fileInfo.IsDir() {
```

Comment:
nit: maybe refactor this block to two separate if-conditions.

Reply:
Addressed in 5fe1126.